mysite/templates/tournament.html

```diff
@@ -234,12 +234,13 @@ <h2 align="center">Wait for your opponent to submit bans, and then Refresh!</h2>
 <label for="form-2" style="font-size: 20px;">How many battles did you win:</label>
     <input type="hidden" name="form_type" value="form2">
     <select name="form-type" id="form2" style="font-size: 20px; padding: 10px;">
-      <option value="0">0</option>
       {% if opponent != user %}
+      <option value="0">0</option>
       <option value="1">1</option>
       <option value="2">2</option>
-      <option value="3">3</option>
```

Comment:
Since we anyways handle win counts separately in the round_wins counting part in python, should we let this be? I forgot if/how html is telling python how someone got a bye, or whether we are reading it at new round generation